content/en/user-manual/optimization/load-time.md

```diff
@@ -4,7 +4,9 @@ template: usermanual-page.tmpl.html
 position: 5
 ---
 
-Optimizing isn't just related to improving frame rate. Fast load times are also critical. The faster your app loads, the more likely your users will stick around to experience it. Here are some tips to achieve super-fast load times:
+Optimizing isn't just related to improving frame rate. Fast load times are also critical. The faster your app loads, the more likely your users will stick around to experience it. For example, users on mobiles for browser based games will wait 4-5 secs of inactivity before leaving.
```

Comment:
Could that claim link to a citation?

Reply:
The only links I have are for websites: https://wemakewebsites.com/blog/improve-page-load-speed-increase-conversion/

Or from Facebook: https://developers.facebook.com/docs/games/instant-games/guides/game-performance/

If we don't mind linking to Facebook, I can rephrase and make it fit?

Reply:
Instead of making a claim, I would just make a recommendation. `Aim to have your app load in less than 5 seconds to prevent users from churning.`

Reply:
Yeah, that sounds good